samples/apps/bookinfo/rules/rule-changes.md

```diff
@@ -0,0 +1,250 @@
+## Config Model Rule Changes
+
+The following rule resource changes are needed to migrate
+from Istio 0.1 (alpha) to Istio 0.2 config format.
+
+All of the 0.2 Pilot config property names are now aligned with the attibute vocabulary
+used for Mixer config. The unified config model design can be found [here](https://docs.google.com/document/d/1fGZpgFWJZhNRlQoBlCW815aOFR-ewfxKhla0CcPRsBM/edit#).
+
+### Create Route Rule
+
+0.1.x:
+```
+istioctl create route-rule -f myrule.yaml
+```
+0.2.x:
+```
+kubectl apply -f myrule.yaml
+```
```

Comment:
Let's remove this piece please. Maybe even axe this document or move to pilot.

Reply:
Move in another PR works for you?

Reply:
Yep.

Reply:
I will clean this up once we resolve https://github.com/istio/istio/issues/652. I also think that maybe this whole doc should be somewhere else (maybe part of the docs in istio.io?). Maybe this kind of 0.1 to 0.2 migration info should go into the istio blog? Thoughts?